js/index.js

```diff
@@ -37,6 +37,99 @@ const siteContent = {
   },
 };
 
+const navItems = document.querySelectorAll('nav a');
+navItemsArr = Array.from(navItems);
+htmlNavItems = Object.keys(siteContent.nav).map(function(value) {
+  return siteContent.nav[value];
+});
+
+for (var i = 0; i < navItemsArr.length; i++) {
+  navItemsArr[i].textContent = htmlNavItems[i];
+  navItemsArr[i].style.color = "dodgerblue"
+};
+
+const newButton = document.createElement('a')
+newButton.textContent = "Stuff";
+const lastButton = document.createElement('a')
+lastButton.textContent = "BUTTON"
+const navi = document.querySelector("nav")
+navi.appendChild(lastButton)
+navi.prepend(newButton);
+
+const ctah1 = document.querySelector('.cta-text h1');
+ctah1.textContent = siteContent.cta.h1;
+
+const ctaButton = document.querySelector('.cta-text button');
+ctaButton.textContent = siteContent.cta.button;
+
+const ctaImg = document.querySelector('#cta-img');
+ctaImg.src = siteContent.cta["img-src"];
+
+const htmlContent = siteContent['main-content'];
+
+const contentH4 = document.querySelectorAll('h4');
```

Comment:
Very nice creative solution!. This is just preference, you can indent the *if* to look something similar to this.
*
const htmlContentP = Object.keys(htmlContent).map(function(string) {
  if (string.includes('content')) {
    return htmlContent[string];
  }
}).filter(function (el) {
  return el != null;
});
*

Reply:
*
const htmlContentP = Object.keys(htmlContent).map(function(string) {
  if (string.includes('content')) {
    return htmlContent[string];
  }
}).filter(function (el) {
  return el != null;
});
*